src/tweaksConfig.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * This configuration is used as an incubator for experimental features. Tweaks are not officially
+ * supported and are not guaranteed to be stable, i.e. their naming, functionality and API can
+ * change at any time within the tweaks or when being promoted to an official feature and moved
+ * into its final configuration namespace.
+ */
+export interface TweaksConfig {
+  /**
+   * The frequency in seconds onTimeChanged is called with TimeChangedEvents.
+   * Default is 1. Minimum is 0.025.
+   *
+   * @platform iOS, Android
+   */
+  timeChangedInterval?: number;
+  /**
+   * If enabled, HLS playlists will be parsed and additional features and events are enabled. This includes:
+   *
+   * - MetadataEvents carrying segment-specific metadata for custom HLS tags, like #EXT-X-SCTE35
+   * - MetadataParsedEvents carrying segment-specific metadata for custom HLS tags, like #EXT-X-SCTE35
+   * - DrmDataParsedEvents when a #EXT-X-KEY is found
+   * - Player.availableVideoQualities includes additional information
+   * - Automatic retries when HLS playlist requests failed with non-2xx HTTP status code
+   *
+   * Default is false.
+   *
+   * @platform iOS
+   */
+  isNativeHlsParsingEnabled?: boolean;
+  /**
+   * If enabled, playlists will be downloaded by the Bitmovin Player SDK instead of AVFoundation.
+   * This enables additional features and events, like:
+   *
+   * - DownloadFinishedEvents for playlist downloads.
+   * - SourceWarningEvents when no #EXT-X-PLAYLIST-TYPE is found If set to false, enabling
+   * nativeHlsParsingEnabled won’t have any effect.
+   *
+   * Default is true.
+   *
+   * @platform iOS
+   */
+  isCustomHlsLoadingEnabled?: boolean;
+  /**
+   * The threshold which will be applied when seeking to the end in seconds. This value will be used
+   * to calculate the maximum seekable time when calling player.seek(time:) orplayer.playlist.seek(source:time:),
```

Comment:
```suggestion
   * to calculate the maximum seekable time when calling player.seek(time:) or player.playlist.seek(source:time:),
```

Reply:
Regarding the default value, I think it's only a matter of making each platform's default value explicit on the docs. Because since the lib doesn't set any default values itself, this should still be handled by the SDKs natively.

Reply:
Also, the typo in this comment is fixed: https://github.com/bitmovin/bitmovin-player-react-native/pull/40/commits/59d57708d89c0faf4f5d5aca0a42ea2399153f98